app-editors/atom-bin/atom-bin-1.55.0-r1.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop xdg
+
+DESCRIPTION="A hackable text editor for the 21st Century"
+HOMEPAGE="https://atom.io/"
+SRC_URI="https://github.com/atom/atom/releases/download/v${PV}/atom-amd64.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="alsa cups ssl test X"
+RESTRICT="!test? ( test )"
+
+S="${WORKDIR}/atom-${PV}-amd64"
+
+RDEPEND="
+	app-accessibility/at-spi2-atk
+	app-accessibility/at-spi2-core
+	app-crypt/libsecret
+	dev-libs/atk
+	dev-libs/nss
+	alsa? ( media-libs/alsa-lib )
+	cups? ( net-print/cups )
+	ssl? ( dev-libs/openssl )
+	X? (
+		x11-libs/cairo
+		x11-libs/gdk-pixbuf
+		x11-libs/gtk+
+		x11-libs/libX11
+		x11-libs/libxcb
+		x11-libs/libXcomposite
+		x11-libs/libXcursor
+		x11-libs/libXdamage
+		x11-libs/libXext
+		x11-libs/libXfixes
+		x11-libs/libXi
+		x11-libs/libxkbfile
+		x11-libs/libXrandr
+		x11-libs/libXrender
+		x11-libs/libXScrnSaver
+		x11-libs/libXtst
+		x11-libs/pango
+	)
+"
+
+QA_PREBUILT="/opt/atom/*"
+
+src_prepare(){
+	default
+
+	# Remove useless license files.
+	rm LICENSE LICENSES.chromium.html version
+}
+
+src_install(){
+	insinto /opt/"${PN}"
+	doins -r "${S}"/*
+	dosym ../../opt/"${PN}"/atom "${EPREFIX}"/usr/bin/atom
+	fperms +x /opt/"${PN}"/atom
+	dosym ../../opt/"${PN}"/resources/app/apm/bin/apm "${EPREFIX}"/usr/bin/apm
+	fperms +x /opt/"${PN}"/resources/app/apm/bin/apm
+	dosym ../../opt/"${PN}"/resources/app/apm/bin/node "${EPREFIX}"/usr/bin/node
+	fperms +x /opt/"${PN}"/resources/app/apm/bin/node
+	dosym ../../opt/"${PN}"/resources/app/apm/bin/npm "${EPREFIX}"/usr/bin/npm
```

Comment:
are you sure this isn't conflicting with nodejs[npm]?

Reply:
I don't know as I don't use nodejs

Reply:
Is it bundling the while nodejs? How can it run otherwise?